src/rpc/blockchain.cpp

```diff
@@ -191,7 +191,7 @@ static UniValue getbestblockhash(const JSONRPCRequest& request)
     return chainActive.Tip()->GetBlockHash().GetHex();
 }
 
-void RPCNotifyBlockChange(bool ibd, const CBlockIndex * pindex)
+void RPCNotifyBlockChange(MAYBE_UNUSED bool ibd, const CBlockIndex * pindex)
```

Comment:
`ibd` seems to *always* be unused here. I think a better approach is to just remove the argument name